packages/subgraph/src/mappings/cfav1.ts

```diff
@@ -95,13 +95,15 @@ export function handleFlowUpdated(event: FlowUpdated): void {
     const flowRateDelta = flowRate.minus(oldFlowRate);
     const isCreate = oldFlowRate.equals(BIG_INT_ZERO);
     const isDelete = flowRate.equals(BIG_INT_ZERO);
+
     if (isDelete) {
         const streamRevision = getOrInitStreamRevision(
             senderAddress,
             receiverAddress,
             tokenAddress
         );
         streamRevision.revisionIndex = streamRevision.revisionIndex + 1;
+
```

Comment:
Is it still pointing to the previous stream here?

Reply:
yeah, `StreamRevision` will always point to the "most recent alive" stream

Reply:
It's a bit weird. There will be a revision index, for example "5", but the stream being pointed at is "4", as long as the "5" is not created yet. Maybe rename to `recentStream` on `StreamRevision` if this behaviour is wanted.

Reply:
fair, that is more descriptive